rabbit_context.py

```diff
@@ -8,22 +8,26 @@
 class RabbitContext:
 
     def __init__(self, **kwargs):
-        self._host = kwargs.get('host') or os.getenv('RABBITMQ_SERVICE_HOST', 'localhost')
-        self._port = kwargs.get('port') or os.getenv('RABBITMQ_SERVICE_PORT', '5672')
+        self._host = kwargs.get('host') or os.getenv('RABBITMQ_SERVICE_HOST', 'rabbitmq')
```

Comment:
This host should be `localhost` instead of `rabbitmq`

Reply:
@ryangrundy7 defaulting to dev (localhost) config is unusual. Is this just for getting the tests to pass locally?

Reply:
I was running the sample loader and it was getting `socket.gaierror: [Errno 8] nodename nor servname provided, or not known` with it as `rabbitmq`. I was running the `pipenv run python` command from the readme

Reply:
Hmm. We should probably add the required environment variables to the readme in that case.